python/mdvtools/dbutils/mdv_server_app.py

```
@@ -738,79 +1173,344 @@
+                # Step 5: Get data from the POST request
+                data = request.get_json()
+                target_user_id = data.get('user_id')  # User to be added
+                permission = data.get('permission', 'view')  # "view", "edit", or "owner"
+
+                if not target_user_id or permission not in ["view", "edit", "owner"]:
+                    return jsonify({"error": "Invalid user or permission"}), 400
+                
+                # Step 4: Determine permission flags
+                is_owner = permission == "owner"
+                can_write = permission in ["edit", "owner"]
+                can_read = True  # Always true if added to a project
```

Ruff (0.8.2)

Local variable `can_read` is assigned to but never used

Remove assignment to unused variable `can_read`

(F841)